app/assets/javascripts/application.js

```diff
@@ -16,3 +16,11 @@
 //= require_tree .
 
 $(function(){ $(document).foundation(); });
+
+function toggleVisibility(id) {
```

Comment:
Put this function in its own js file that is descriptive about what it does. Avoid putting additional scripts directly into `application.js`.